### ISSUE TITLE
Accessibility Checklists: Create Input Mask checklist

### ISSUE THREAD

**Issue by brunerae** [uswds/uswds-site#2625]
Follow-up from https://github.com/uswds/uswds/issues/5743

### A11Y
- [x] Review checklist content
- [x] Final approval of checklist content

### Content
- [x] Draft checklist content and send for review with A11Y
- [x] Final edits to checklist content
- [x] Include copy about clear functionality/icon applied by browsers (see [Dev Sync notes](https://docs.google.com/document/d/172TU5BVS9kGZQDuPcAhGoZjaTd2ngzcTayq4WRRTBUs/edit)) 

### Dev
- [x] #2627

## References
[Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing)
[List of all components](https://docs.google.com/spreadsheets/d/1ASzZ80oI6WVWAifqPO6gYAqrsRaiZs_qGTOStuQPyKs/edit?usp=sharing)

**Comment by sarah-sch** [comment]
Text has been drafted. @amycole501, @alex-hull, and @amyleadem, let's discuss whether this one is ready for a page draft. There is one failure noted in the test sheet.
